### ISSUE TITLE
Some questions about the test sets.

### ISSUE THREAD

**Issue by NabsTao** [hujunxianligong/Tensorflow-CNN-Tutorial#5]
Thank you for all your assistance.

Is the format of test sets same to the train set?

Are their naming method same? 

Are there some requirements of test sets? Such as the number of pictures in test sets.

**Comment by lighTQ** [comment]
in  line 'for fpath, real_label, predicted_label in zip(fpaths, labels, predicted_labels_val): '
the 'fpaths ' only show the truncated ops filenames  and not the testdataset filename,so it's not necessary
